src/App.vue

```diff
@@ -227,7 +256,21 @@ export default {
 			await this.$nextTick()
 			this.onWindowResize()
 		}
-		this.schedule.tracks.forEach(t => { t.value = t.id; t.label = getLocalizedString(t.name); this.allTracks.push(t) })
+		this.filter.tracks.data = this.schedule.tracks.map(t => { t.value = t.id; t.label = getLocalizedString(t.name); return t })
```

Comment:
You are generating `filter.tracks` from `schedule.tracks` and also modify `schedule.tracks` at the same time. It is prone to bug. Please change in one of these two ways:

- Move to actions of modifying  `schedule.tracks` to when `schedule` or `schedule.tracks` is populated.

- Copy the elements of `schedule.tracks` and add new properties to the copy only.

Reply:
Hi @hongquan, I have fixed as your suggestion, I create a copy of schedule.tracks then add new properties on it only, add new properties on current schedule.tracks is not necessary so I removed it.